### ISSUE TITLE
Double-Click Username to Reply function not working (again)

### ISSUE THREAD

**Issue by AlcaDesign** [Tester798/ffzenhancing#25]
The option "Double-Click Username to Reply" no longer works (again).

Error in the console:
```
Uncaught TypeError: Cannot read properties of undefined (reading 'value')
    at getChatInput (ffzenhancing.js:372:102)
    at HTMLBodyElement.<anonymous> (ffzenhancing.js:1003:27)
```
https://github.com/Tester798/ffzenhancing/blob/258548bcc084232b72ae7d48b3a2bb1319c791de/ffzenhancing.js#L372

**Comment by Tester798** [comment]
It works for me.
Do you have enabled new chat input box which replaces emote codes with images?

**Comment by AlcaDesign** [comment]
I keep disabling the WYSIWYG chat input experiment (it keeps getting re-enabled?) because of the increased input latency. It can't keep up with my incredible typing powers ![](https://static-cdn.jtvnw.net/emoticons/v2/25/default/dark/1.0). I have access to a thousand+ emotes (plus emoji get thrown in), so the autocomplete is painful and sometimes picks the wrong emote, and more issues like copy-pasting. Fair enough if this feature doesn't work with that preference. Hopefully FFZ (or a lovely extension) will support disabling it outright or Twitch improves its performance greatly.

**Comment by Tester798** [comment]
Indeed, it works slowly for me too, I also hope it will be fixed in the future.
For now you can probably use version 6.74 as a userscript if you want.

**Comment by Tester798** [comment]
Ok, this should be fixed in 6.80.
